src/hotspot/share/runtime/objectMonitor.hpp

```diff
@@ -390,6 +390,7 @@ class ObjectMonitor : public CHeapObj<mtObjectMonitor> {
   bool      enter(JavaThread* current);
   bool      try_enter(JavaThread* current, bool check_for_recursion = true);
   bool      spin_enter(JavaThread* current);
+  void      notify_contended_enter(JavaThread *current);
```

Comment:
This helper method should be private. Thanks

Reply:
Thanks. Fixed now.